src/test/java/org/takes/facets/auth/codecs/CcBase64Test.java

```diff
@@ -134,9 +134,24 @@ public void decodesNonBaseSixtyFourAlphabetSymbols() throws IOException {
      * @throws Exception If some problem inside
      */
     @Test
-    public void equalsAndHashCodeEqualTest() throws Exception {
-        EqualsVerifier.forClass(CcBase64.class)
-            .suppress(Warning.TRANSIENT_FIELDS)
-            .verify();
+    public void equalsTest() throws Exception {
+        new Assertion<>(
+            "CcBase64 objects must be equal",
+            new CcBase64(new CcPlain()),
+            new IsEqual<>(
+                new CcBase64(new CcPlain())
+            )
+        ).affirm();
+    }
+
+    @Test
+    public void hashCodeTest() throws Exception {
+        new Assertion<>(
+            "Hash codes must be equal",
+            new CcBase64(new CcPlain()).hashCode(),
+            new IsEqual<>(
```

Comment:
@fabriciofx put these 3 lines in the one line.

Reply:
@fabriciofx Same as [here](https://github.com/yegor256/takes/pull/995#discussion_r347507143)

Reply:
@fabriciofx we can